scanapp/tasks.py

```diff
@@ -57,6 +59,20 @@ def scan_code_async(url, scan_id, path, file_name):
 
 
 @app.task
+def handle_special_urls(url, scan_id, path, host):
+    """
+    Create and initialise the git repository at a certain path and clone the git repo using `url`
+    and then get the scan done.
+    """
+    if host == 'github.com':
+        logger = logging.getLogger(__name__)
+        logger.info('git repo detected')
```

Comment:
a  log entry should contain more details, e.g. information about the args used (here the host at least)